### PR TITLE
add: datepicker plugin and Flowbite

### PR DESCRIPTION
**Pull request by zoltanszogyenyi**
| Name                 | Link                 |
| -------------------- | -------------------- |
| Tailwind Datepicker| https://github.com/themesberg/tailwind-datepicker |

A datepicker component built with Vanilla JavaScript and Tailwind utility classes.

---

| Name                 | Link                 |
| -------------------- | -------------------- |
| Flowbite| https://flowbite.com/docs/getting-started/introduction/ |

Open-source web components and interactive elements built with Tailwind CSS

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Review by innocenzi — CHANGES_REQUESTED** [review]
Hey, please place your new items below the corresponding emoji groups instead of above, and review the descriptions to follow the [guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md#formats-naming-conventions-and-descriptions). I'll merge after that. :)

**Comment by zoltanszogyenyi** [comment]
@innocenzi I've moved the new additions below and also updates the description according to the guideline. I hope it's good now.

File: README.md [hunk 1]
```diff
@@ -232,10 +232,10 @@
 
 - 💙🧩 [Tailwind UI](https://tailwindui.com) - Component library made with Tailwind CSS.
 - 💙📚 [Headless UI](https://github.com/tailwindlabs/headlessui) - Completely unstyled, fully accessible UI components.
-- 📚 [Flowbite](https://flowbite.com/docs/getting-started/introduction/) - Open-source web components and interactive elements built with Tailwind CSS
 - 📚 [VueTailwind](https://github.com/alfonsobries/vue-tailwind) - Vue.js UI library using Tailwind CSS.
 - 📚 [Tailwind Elements](https://tailwind-elements.com/) - Huge collection of free components, mobile-friendly thanks to Bootstrap 5.
 - 📚 [Vechai UI](https://www.vechaiui.com/) - High-quality accessible React components with the built-in dark mode using Tailwind CSS.
+- 📚 [Flowbite](https://flowbite.com/docs/getting-started/introduction/) - Open-source component library built with Tailwind CSS
```
Comment (innocenzi) [inline README.md:238]:
```suggestion
- 📚 [Flowbite](https://flowbite.com/docs/getting-started/introduction/) - Open-source component library built with Tailwind CSS.
```

File: README.md [hunk 2]
```diff
@@ -131,6 +131,7 @@
 - 🧩 [CSS Variables](https://github.com/omarkhatibco/tailwind-css-variables) - Exports configuration to CSS Custom Properties.
 - 🧩 [CSS Variables](https://github.com/mertasan/tailwindcss-variables) - Exports custom CSS variables (Dark Mode supported).
 - 🧩 [Perspective](https://github.com/Kamona-WD/tailwindcss-perspective) - Adds `perspective` utilities.
+- 🧩 [Tailwind Datepicker](https://github.com/themesberg/tailwind-datepicker) - Adds a datepicker component built with Tailwind CSS and Vanilla JavaScript
```
Comment (innocenzi) [inline README.md:134]:
```suggestion
- 🧩 [Tailwind Datepicker](https://github.com/themesberg/tailwind-datepicker) - Adds a datepicker component built with Tailwind CSS and vanilla JavaScript.
```

**Comment by innocenzi** [comment]
For some reason I can't commit these two changes myself, so please do so yourself and I'll merge after that!

**Comment by zoltanszogyenyi** [comment]
I've resolved your changes, can you merge now?

**Comment by innocenzi** [comment]
You didn't make the modifications I asked, you just resolved (read: dismissed) the two review comments. :/

**Comment by zoltanszogyenyi** [comment]
I've made the changes and also moved the Tailwind Datepicker to the "components" section since it wasn't actually a plugin but a component/library.

PS: I thought that by using the Github "resolve" mechanism it would apply your changes.

**Review by innocenzi — APPROVED** [review]

**Comment by innocenzi** [comment]
> PS: I thought that by using the Github "resolve" mechanism it would apply your changes.

Nope, there's usually a "commit" button next to it for this purpose, but I couldn't see it this time. Anyway, it's merged, thanks!

**Comment by joshmanders** [comment]
Thanks @zoltanszogyenyi!

**Pull request merged**